tools/regression_rllib.py

```diff
@@ -109,6 +109,7 @@ def run_experiment(log_path, experiment_name, training_iteration=100):
     analysis = tune.run(
         "PPO",
         name=experiment_name,
+        mode="max",
```

Comment:
Given no `metric` attribute in `tune.run()`, what is `mode=max` maximizing here?

Reply:
This was being slightly aggressive with adding default `mode` in case of analysis being used since I was seeing related assertions.